index.html

```diff
@@ -8,29 +8,149 @@
+        <div class="text-card">
+        <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+          <ul class="courses">
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+          </ul>
+        <button type="submit">See Project</button>
+        </div>
+        </div>
+        <div class="item">
+        <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+        <div class="text-card">
+        <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+          <ul class="courses">
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+          </ul>
+        <button type="submit">See Project</button>
+        </div>
+        </div>
+        <div class="item"><img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+        <div class="text-card">
+        <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+          <ul class="courses">
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+          </ul>
+        <button type="submit">See Project</button>
+        </div></div>
+        <div class="item"><img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+        <div class="text-card">
+        <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+          <ul class="courses">
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+          </ul>
+        <button type="submit">See Project</button>
+        </div></div>
+        <div class="item"><img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+        <div class="text-card">
+        <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+          <ul class="courses">
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+          </ul>
+        <button type="submit">See Project</button>
+        </div></div>
+        <div class="item"><img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+        <div class="text-card">
+        <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+          <ul class="courses">
+          <li>Ruby on Rails</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+          </ul>
+        <button type="submit">See Project</button>
+        </div></div>
+      </article>
+    </section>
+    <section id="about-me">
+      <article id="about-article">
+        <h1>About me</h1>
+        <p id="about-text"> I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button type="submit">Get My Resume</button>
+      </article>
+      <article class="about-row">
+        <div class="about-item">
```

Comment:
- [ ] I kindly suggest that you make the containers of the nested technology categories("courses-about") a `ul` and use an `li` for each Language or Framework("about-item")
- [ ] For a list of items, always use the unordered list or `ul` tag and each list item can then be wrapped in a `li`. The list of icons thus should be wrapped in a `ul` tag instead of a `div` to maintain its semantic meaning. Kindly alter this code to use an unordered list tag instead.

Reply:
Sorry to bother you again @topeogunleye. The list of icons you mentioned, what line?

I have already changed the "about-row" article to a UL tag while changing all its direct children as well to LI.
However, I can't find the icons you mean.

Reply:
No icons. I meant languages and frameworks pls

Reply:
Thank you bro @topeogunleye